### ISSUE TITLE
Design new icon for the document-only toggle

### ISSUE THREAD

**Issue by pedropintosilva** [CollaboraOnline/online#9062]
For the big picture please consult #9059  

----

Please feel free to discuss and share your opinions both regarding the label and the overall icon's metaphor

Before thinking about the icon it might be helpful to decide on the label first and how it relates to the icon. Initially I thinking on something along the lines of "Switch background" but not it seems inappropriate for a toggle (a toggle has 2 states, with that label it's perceived as a simple button/action).  So, probably something that could be more specific to color and that would pair nicely with a document/page icon could be: "Inverted Colors" or something similar. Let me lay out the behaviour so the design becomes easier:

* The new toggle appears only when dark mode is on
  * By default the status should be on (pressed)
* The new toggle acts solely on the document and not in the UI

In summary, an icon that revolves a) around a document or page would be good or b) around contrast icon or c) both

**Comment by pedropintosilva** [comment]
https://github.com/CollaboraOnline/online/pull/9112
